docs/CUSTOM-VALUES.md

```diff
@@ -0,0 +1,59 @@
+# Custom values
+
+Fabric API modules may have certain custom values set or check for the presence of specific custom values.
+
+The use of custom values ranges from enabling/disabling features to extra metadata about fabric api modules.
+
+# Provided by Fabric API
+
+Custom values provided by Fabric API are for use by other mods.
+
+## `fabric-api:module-lifecycle`
+
+Specifies the lifecycle of a fabric api module.
+This may be used by other mods to determine 
```

Comment:
Determine what? Seems to be some text missing here.

Reply:
Okay now I have determined the answer